pkg/test/framework/features/features.yaml

```diff
@@ -30,9 +30,16 @@ features:
   # features that allow users to secure their services and service mesh.
   security:
     peer:
+      secure-naming:
     user:
     control-plane:
       k8s-certs:
+      mtls:
+      tls:
+      plugin-cert:
+    webhook:
+    chiron:
```

Comment:
"chiron" is not a feature, its an internal name. The feature is "k8s-certs" I think

Reply:
I think it is required as we design it the test of chiron should be added a label of chiron

Reply:
yes remove it